component/libfuse/libfuse_handler.go

```diff
@@ -185,6 +185,10 @@ func libfuse_init(conn *C.fuse_conn_info_t, cfg *C.fuse_config_t) (res unsafe.Po
 	conn.want |= C.FUSE_CAP_PARALLEL_DIROPS
 	conn.want |= C.FUSE_CAP_AUTO_INVAL_DATA
 	conn.want |= C.FUSE_CAP_READDIRPLUS
+	conn.want |= C.FUSE_CAP_ASYNC_READ
+
+	conn.max_background = 128
```

Comment:
could we add comments elaborating here please

Reply:
added comments for all the flags